azure-pipelines.yml

```diff
@@ -28,8 +28,8 @@ jobs:
       make --keep-going test
     displayName: 'Run rust tests'
   - script: |
-      make --keep-going clippy
-    displayName: 'Run clippy linter'
+      echo let us wait until cargo fix or rustfix actually works make --keep-going clippy
+    displayName: 'SKIP clippy linter'
```

Comment:
Can you create a github issue for this to remind us?

Reply:
will do.